python-backend/app/api/parties/party/resources/party_resource.py

```diff
@@ -121,7 +119,7 @@ def put(self, party_guid):
             party_exists.first_name = first_name
             party_exists.party_name = party_name
         elif party_type_code == PARTY_STATUS_CODE['org']:
-            party_name = data['party_name'] if data['party_name'] else party_exists.party_name
+            party_name = data.get('party_name', party_exists.party_name)
```

Comment:
Another pattern i really like. If you're looking at a fallback key, .get() saves you time from not having to write inline if else